src/ui/React/ImportSave/ImportSave.tsx

```diff
@@ -124,14 +125,29 @@ export const ImportSave = (props: { saveData: SaveData; automatic: boolean }): J
   useEffect(() => {
     async function fetchData(): Promise<void> {
       const dataBeingImported = await saveObject.getImportDataFromSaveData(props.saveData);
-      const dataCurrentlyInGame = await saveObject.getImportDataFromSaveData(await saveObject.getSaveData(true));
+      let saveData;
+      try {
+        saveData = await saveObject.getSaveData(true);
+      } catch (error) {
+        return Promise.reject(error);
```

Comment:
I think this is equivalent to throwing `error`? In which case, I don't see the point in adding try/catch here at all.